docs/Developers/Git_Basics.rst

```diff
@@ -110,6 +122,15 @@ It will give you an idea about what files are currently modified.
 Adding the files and commiting
 ------------------------------
 
+First, make sure you're on the correct branch and not developing on master! If
+you've been following this guide, and this is your first pull request,
+you should be developing on the "my-first-good-pull-request" branch.
+You can check your branch with:
+
+::
+
+    $ git status
+
```

Comment:
Yes, but the issue seemed to ask for periodic checks, should I still remove this?